rfc-099-stack-overflow-for-gds.md

```diff
@@ -0,0 +1,41 @@
+# Q&A Forum for GDS developers
+
+## Summary
+
+A searchable, long-lived body of developer knowledge would make finding
+answers easier.
+
+StackOverflow is a Q&A forum that all developers at GDS are familiar with.
+A private instance of StackOverflow would ensure that valuable knowledge
+is retained and more easily found.
+
+## Problem
+
+When faced with a problem we're not able to solve, we ask for help from
```

Comment:
We (in govuk) also maintain [significant developer docs](https://docs.publishing.service.gov.uk/), however it's an intermingling of background documentation and answers to specific issues, so it would probably be of benefit to have a different system for the latter type.

It also has bad / weird search.